### PR TITLE
Bug 1932443: Fix missing metrics port in manifests

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### Description
Expose ports for `http` and `cr` metrics in the ClusterServiceVersion deployment manifest for the cluster-logging-operator.
Fix generate bundle command
Generate bundle.

/cc @periklis 
/assign @jcantrill 

/cherry-pick 

### Links

- Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1932443

### CODE REVIEW

**Review by periklis — CHANGES_REQUESTED** [review]
Needs a run `make bundle` and include the bundle copy of the CSV.

**Comment by vparfonov** [comment]
> Needs a run `make bundle` and include the bundle copy of the CSV.
@periklis `make bundle` target not found for CLO

**Comment by vparfonov** [comment]
> > Needs a run `make bundle` and include the bundle copy of the CSV.
> > @periklis `make bundle` target not found for CLO

Looks like need to run `make generate-bundle`

File: manifests/4.6/cluster-logging.v4.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -315,6 +315,11 @@ spec:
                 imagePullPolicy: IfNotPresent
                 command:
                 - cluster-logging-operator
+                ports:
+                  - containerPort: 8383
+                    name: http-metrics
+                  - containerPort: 8686
+                    name: cr-metrics
```
Comment (periklis) [inline manifests/4.6/cluster-logging.v4.6.0.clusterserviceversion.yaml:322]: Have you tested this on a test cluster? Do the SVCs use the ports by name or by port-number directly? If by name, do the names match between here and SVCs?

**Review by periklis — COMMENTED** [review]
In general looks good to me. I suggest someone from log core takes also a look as the extra changes in the bundle directory indicate that the bundle was not sync'ed for a long time. I am not sure if the old state was delivered to customer clusters already. If so we would break compatibility with changes like CLF from cluster-scoped to namespace-scoped.

File: manifests/4.6/cluster-logging.v4.6.0.clusterserviceversion.yaml (hunk 1 above)
Comment (vparfonov) [reply manifests/4.6/cluster-logging.v4.6.0.clusterserviceversion.yaml:322]: Yes, i tested it. All deployed normally. Service use port-number directly.

**Review by periklis — APPROVED** [review]
/lgtm

**Pull request closed without merging**